src/main/java/mat/server/service/impl/FhirMeasureServiceImpl.java

```diff
@@ -71,31 +86,67 @@ public FhirConvertResultResponse convert(ManageMeasureSearchModel.Result sourceM
             // If there is no FHIR CQL, then we don't persist the measure. FHIR measure cannot be created.
             throw new MatException("Your measure cannot be converted to FHIR. Outcome: " + validationStatus.getOutcome() + " Error Reason: " + validationStatus.getErrorReason());
         } else {
-            persistFhirMeasure(loggedinUserId, fhirConvertResultResponse, sourceMeasureDetails, fhirCqlOpt);
+            persistFhirMeasure(loggedinUserId, fhirConvertResultResponse, sourceMeasureDetails, fhirCqlOpt.get());
         }
 
         return fhirConvertResultResponse;
     }
 
+    /**
+     * The only purpose of this is to make it mockable.
+     * setMeasureXMLAsByteArray has Hibernate LOB crap in it that is difficult to mock away here.
```

Comment:
Recommend "...has Hibernate LOB in it..."

Reply:
lol, fixed